frontend/footer_info.py

```diff
@@ -0,0 +1,139 @@
+import streamlit as st
+
+# Connect file to fonts.css
+with open("css/fonts.css") as f:
+    st.markdown(
+        f"""<style>{f.read()}</style> <body>
+
+  </body>""",
+        unsafe_allow_html=True,
+    )
+
+# Styling
+st.markdown("""
```

Comment:
if possible, maybe we can just aggregate all of the css into separate css file to be consistent, since part of our css code is in style.css